tests/functional_tests/iterator_tests/test_unroll_reduce.py

```diff
@@ -96,6 +98,21 @@ def reduction_with_irrelevant_full_shift():
     )
 
 
+@pytest.mark.parametrize(
+    "reduction",
+    [
+        "basic_reduction",
+        "reduction_with_irrelevant_full_shift",
+        "reduction_with_shift_on_second_arg",
+    ],
+)
+def test_get_partial_offsets(reduction, request):
+    offset_provider = {"dim": SimpleNamespace(max_neighbors=3, has_skip_values=False)}
```

Comment:
```suggestion
    offset_provider = {"FooDim": SimpleNamespace(max_neighbors=3, has_skip_values=False)}
```
Just a suggestion. The non-capitalization doesn't look nice.

Reply:
Replaced by capitalized names